### PR TITLE
Performance: testing command

### PR DESCRIPTION
**Pull request by rowanseymour**
This PR replaces the apiaudit command and the perf_tests unit test module which haven't been used very much, with a more generic performance testing script designed to be run on a database generated with the new `make_test_db` command.

```
./manage.py perf_test                        # test all URLs
./manage.py perf_test  --include=/api/v2/*   # test just the APIv2 URLs
```

Each URL is hit 3 times and we display the min and max times, e.g.

screen shot 2017-02-20 at 17 15 26

Haven't included message stuff yet and messages aren't yet generated by `make_test_db`.

### CODE REVIEW

**Comment by nicpottier** [comment]
Nice, this will be way more reproducible.

Seems to me anything over a second should be read, with maybe 250ms being yellow?

**Comment by nicpottier** [comment]
Although probably only for API calls. Guess user triggered stuff can be slower.

**Comment by rowanseymour** [comment]
I need to spend a bit more time investigating but it looks like all of the non-API calls take at least ~1.7 secs to return. Could template parsing, middleware... not sure.

How about warning is 1+ second, fail is 2+ seconds until we get a better grip on what's happening here? I do agree tho that the goal should be sub second for everything.

**Comment by nicpottier** [comment]
Ah right, are these running with collect static and compress? I seem to remember our dev server has crazy latency when not in prod mode and it is related to that.

**Comment by rowanseymour** [comment]
@nicpottier running after collectstatic+compress makes a huge difference and shows where the real problems are:

screen shot 2017-02-20 at 20 43 53

(changed warning threshold to 500ms and failure to 1 sec)

File: temba/utils/management/commands/perf_test.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+DEFAULT_RESULTS_FILE = '.perf_results'
+
+# allow this maximum request time
+ALLOWED_MAXIMUM = 3
+
+# allow this much absolute change from previous results (50ms)
+ALLOWED_CHANGE_MAXIMUM = 0.05
+
+# allow this much percentage change from previous results
+ALLOWED_CHANGE_PERCENTAGE = 5
```
Comment (rowanseymour) [inline temba/utils/management/commands/perf_test.py]: @nicpottier I've reworked things a bit to make it clearer what the criteria for a failed test is, and easier to exit with a non-zero code if any test fails. The 3 constants above determine what is acceptable.

**Pull request merged**